app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-specialist-link-tracker.js

```diff
@@ -39,8 +39,8 @@ window.GOVUK.analyticsGa4.analyticsModules = window.GOVUK.analyticsGa4.analytics
         return
       }
 
-      // don't track this link if it's already being tracked by the other link tracker
-      if (element.closest('[data-ga4-link]')) {
+      // don't track this link if it's already being tracked by the another tracker (e.g. the link tracker or ecommerce tracker)
+      if (element.closest('[data-ga4-link]') || element.closest('[data-ga4-ecommerce-path]')) {
```

Comment:
I think you should add a test for this, similar to this one: https://github.com/alphagov/govuk_publishing_components/blob/4caf99cc16296ed07573d23dd6daa5dc650bd1d7/spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-specialist-link-tracker.spec.js#L279